zfmt/ast.go

```diff
@@ -683,8 +683,21 @@ func (c *canon) fromElem(elem *ast.FromElem) {
 	if elem.Args != nil {
 		c.fromArgs(elem.Args)
 	}
-	if elem.Alias != nil {
-		c.write(" %s", sup.QuotedName(elem.Alias.Text))
+	c.tableAlias(elem.Alias)
```

Comment:
Nit: The usual pattern in here is to do the nil check in the caller rather the callee.
```suggestion
	if elem.Alias != nil {
		c.tableAlias(elem.Alias)
	}
```